packages/bot/src/stores/chart-store.js

```diff
@@ -0,0 +1,123 @@
+
+import {
+    action,
+    computed,
+    observable,
+    reaction,
+}                     from 'mobx';
+import { tabs_title } from '../constants/bot-contents';
+import ServerTime     from '../services/api/server_time';
+
+const g_subscribers_map = {};
+let WS;
+
+export default class ChartStore {
+    constructor(root_store) {
+        this.root_store = root_store;
+        WS = root_store.ws;
+        const { run_panel } = root_store;
+
+        reaction(() =>
+            run_panel.is_running, () =>
+            run_panel.is_running ?
+                this.startBot() :
+                this.stopBot()
+        );
+    }
+
+    @observable symbol;
+    @observable is_chart_loading;
+    @observable main_barrier = null;
+    @observable chart_type;
+    @observable granularity;
```

Comment:
Could we set default values for some of these so it's clear as to what they will contain?